TeadsSampleApp/Controllers/InRead/Mopub/ScrollView/InReadMopubScrollViewController.swift

```diff
@@ -26,6 +26,7 @@ class InReadMopubScrollViewController: TeadsViewController {
         let config = MPMoPubConfiguration(adUnitIdForAppInitialization: MOPUB_AD_UNIT_ID)
         bannerView = MPAdView(adUnitId: MOPUB_AD_UNIT_ID)
         bannerView.delegate = self
+        bannerView.frame = CGRect(x: 0, y: 0, width: 350, height: 250);
```

Comment:
I think we can use UIScreen.main.bounds.width as width here

Reply:
I think there is more elegant, use mopub adSize constants

Reply:
here it is: 
```
MPAdapterTeadsConstants.bannerSize
// static let bannerSize = CGSize(width: 320, height: 250)
```

Reply:
I've decided to use `slotView.bounds` rather than constants from our adapter or from MoPub. Constant from MoPub have height only, no width, so that's why I've a weird ratio, width was negative.